cdm/core/src/main/java/ucar/nc2/internal/dataset/conv/DefaultConventions.java

```diff
@@ -213,12 +213,14 @@ protected AxisType getAxisType(VariableDS.Builder vb) {
       {
         return AxisType.Height;
       }
+      logger.info("Unit '" + unit + "' is not compatible with meters for axis '" + vname + "'");
```

Comment:
This class also has a `parseInfo` I could log to but I am not sure where those messages end up